pagopa/pn-frontend-e2e
# Test 9179 GruppiPG

## Short description

Created tests for the creation, modification, suspension and elimination of groups in PG

## List of changes proposed in this pull request

- Created a file to contain all the tests on gruppi for PG
- Created a file with the steps implementations
- Created a file for the actual code

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@TA_gruppiPG"` and verify if the tests work fine.
